src/Symfony/Component/Process/Messenger/ExecuteProcessHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Messenger;
+
+use Symfony\Component\Process\Process;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class ExecuteProcessHandler
+{
+    public function __invoke(ExecuteProcess $message): Process
+    {
+        $process = Process::fromShellCommandline(
+            $message->command,
+            $message->cwd,
+            $message->env,
+            $message->input,
+            $message->timeout,
+        );
+
+        return $process->mustRun();
```

Comment:
Throws an exception if the process fails, should this be configurable?

Reply:
This is actually going to be a problem I think. `ProcessFailedException` (which this line throws on error) can't be serialized because it contains the `Process`.

Like stated in https://github.com/symfony/symfony/pull/49814#issuecomment-1486903588, it is useful to have the output in the exception. Should we create a specific exception for this (`RunProcessFailedException`) that has the exit code, output and error output?